1/Dockerfile.rhel7

```diff
@@ -54,6 +54,43 @@ RUN mkdir -p /opt/openshift/plugins && \
+    ln -s /usr/lib64/jenkins/workflow-cps.hpi /opt/openshift/plugins/workflow-cps.jpi && \
+    ln -s /usr/lib64/jenkins/pipeline-stage-step.hpi /opt/openshift/plugins/pipeline-stage-step.jpi && \
+    ln -s /usr/lib64/jenkins/workflow-basic-steps.hpi /opt/openshift/plugins/workflow-basic-steps.jpi && \
+    ln -s /usr/lib64/jenkins/pipeline-input-step.hpi /opt/openshift/plugins/pipeline-input-step.jpi && \
+    ln -s /usr/lib64/jenkins/workflow-support.hpi /opt/openshift/plugins/workflow-support.jpi && \
+    ln -s /usr/lib64/jenkins/workflow-step-api.hpi /opt/openshift/plugins/workflow-step-api.jpi && \
+    ln -s /usr/lib64/jenkins/workflow-multibranch.hpi /opt/openshift/plugins/workflow-multibranch.jpi && \
+    ln -s /usr/lib64/jenkins/cloudbees-folder.hpi /opt/openshift/plugins/cloudbees-folder.jpi && \
+    ln -s /usr/lib64/jenkins/pipeline-stage-view.hpi /opt/openshift/plugins/pipeline-stage-view.jpi && \
+    ln -s /usr/lib64/jenkins/jquery-detached.hpi /opt/openshift/plugins/jquery-detached.jpi && \
+    ln -s /usr/lib64/jenkins/handlebars.hpi /opt/openshift/plugins/handlebars.jpi && \
+    ln -s /usr/lib64/jenkins/pipeline-rest-api.hpi /opt/openshift/plugins/pipeline-rest-api.jpi && \
+    ln -s /usr/lib64/jenkins/momentjs.hpi /opt/openshift/plugins/momentjs.jpi && \
+    ln -s /usr/lib64/jenkins/workflow-remote-loader.hpi /opt/openshift/plugins/workflow-remote-loader.jpi && \
+    ln -s /usr/lib64/jenkins/scm-api.hpi /opt/openshift/plugins/scm-api.jpi && \
+    ln -s /usr/lib64/jenkins/branch-api.hpi /opt/openshift/plugins/branch-api.jpi && \
+    ln -s /usr/lib64/jenkins/workflow-scm-step.hpi /opt/openshift/plugins/workflow-scm-step.jpi && \
+    ln -s /usr/lib64/jenkins/git.hpi /opt/openshift/plugins/git.jpi && \
+    ln -s /usr/lib64/jenkins/mapdb-api.hpi /opt/openshift/plugins/mapdb-api.jpi && \
+    ln -s /usr/lib64/jenkins/subversion.hpi /opt/openshift/plugins/subversion.jpi && \
+    ln -s /usr/lib64/jenkins/mercurial.hpi /opt/openshift/plugins/mercurial.jpi && \
+    ln -s /usr/lib64/jenkins/matrix-project.hpi /opt/openshift/plugins/matrix-project.jpi && \
+    ln -s /usr/lib64/jenkins/multiple-scms.hpi /opt/openshift/plugins/multiple-scms.jpi && \
+    ln -s /usr/lib64/jenkins/ssh-credentials.hpi /opt/openshift/plugins/ssh-credentials.jpi && \
+    ln -s /usr/lib64/jenkins/pipeline-utility-steps.hpi /opt/openshift/plugins/pipeline-utility-steps.jpi && \
+    ln -s /usr/lib64/jenkins/script-security.hpi /opt/openshift/plugins/script-security.jpi && \
+    ln -s /usr/lib64/jenkins/structs.hpi /opt/openshift/plugins/structs.jpi && \
+    ln -s /usr/lib64/jenkins/git-client.hpi /opt/openshift/plugins/git-client.jpi && \
+    ln -s /usr/lib64/jenkins/git-server.hpi /opt/openshift/plugins/git-server.jpi && \
+    ln -s /usr/lib64/jenkins/plain-credentials.hpi /opt/openshift/plugins/plain-credentials.jpi && \
+    ln -s /usr/lib64/jenkins/ace-editor.hpi /opt/openshift/plugins/ace-editor.jpi && \
```

Comment:
in particular, the fact that the .hpi suffix changes to .jpi makes it a bit non-standard